android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -432,7 +469,12 @@ public void onFinishScan(final ArrayList<ClientScanResult> clients) {
      * return {@code true} if Wi-Fi AP is enabled
      */
     private void isWiFiAPEnabled(Result poResult) {
-        poResult.success(moWiFiAPManager.isWifiApEnabled());
+        try {
+            poResult.success(moWiFiAPManager.isWifiApEnabled());
+        }
+            catch (SecurityException e) {
+                Log.e(WifiIotPlugin.class.getSimpleName(), e.getMessage(), null);
+            }
```

Comment:
Should we also execute a
```
poResult.error(...)
```
in the catch? Otherwise in case of error we are not setting any result I believe.

Reply:
Resolved in 5e480d26f2dbfcd31d2afa8ddc6a83d434d5ea55